ansible/group_vars/all/vars.yml

```diff
@@ -121,25 +121,23 @@ hostapd_password: 1234567890
 host_channel: 8
 host_wifi_mode: g
 driver_name: nl80211
-host_country_code: KE   
+host_country_code: KE
```

Comment:
Are the `host_country_code` and `wifi_country` necessary? note that we have also schools in Tanzania, but I think that in our case they are not used 🤔

Reply:
Wifi Country is correct.Transmitting more power than permitted is illegal and subjected to prosecution.